### PR TITLE
Site migration

### PR DESCRIPTION
**Pull request by ethanresnick**
@adamhooper Please review when you have a sec! I think this is the start of the right setup, though now I have to figure out whatever's involved in actually getting the blog server to start up and listen on 8000

### CODE REVIEW

**Comment by adamhooper** [comment]
Looks good. But we're not going to start the blog on http://localhost:8000. It's already running on http://ip-10-20-194-108.ec2.internal:80.

**Comment by ethanresnick** [comment]
@adamhooper Ok, I removed the commit with all the blog-related stuff now that we decided to do that outside of haproxy. If you want to merge what's left (i.e. simplifications to the `http_redirect` frontend), feel free. Otherwise, if that's more trouble than its worth, feel free to just close this + delete the branch.

File: cloud-init/web.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ write_files:
         mode http
         bind 0.0.0.0:80
         timeout client 50s
-        redirect prefix https://#OVERVIEW_ENVIRONMENT_ADDRESS# code 301 
+        redirect scheme https code 301
```
Comment (adamhooper) [inline cloud-init/web.txt]: This isn't the same thing. It leads to double-redirect: http://overviewproject.org => https://overviewproject.org => https://www.overviewproject.org. Please undo this line.

File: cloud-init/web.txt [hunk 2]
```diff
@@ -42,10 +42,7 @@ write_files:
         capture request header Referer len 200
         capture request header User-Agent len 200
 
-        capture response header Content-Type len 30
-        capture response header Content-Encoding len 10
-        capture response header Cache-Control len 200
-        capture response header Last-Modified len 200
+        capture response header Location len 100
```
Comment (adamhooper) [inline cloud-init/web.txt]: Why this change? IIRC Logstash is expecting our capture stuff to be configured just so.
